### ISSUE TITLE
Support latest node LTS version

### ISSUE THREAD

**Issue by mwelche** [ProjectOpenSea/seaport-js#42]
receiving this error installing opensea-js version 4.0.0-beta.2 via yarn.

```
error @opensea/seaport-js@1.0.0: The engine "node" is incompatible with this module. Expected version "16.11.0". Got "16.14.2"
```

If there aren't compatibility issues, would be nice to support the latest versions of node. 16.15.1 is currently the latest LTS version

**Comment by wong2** [comment]
Yeah, why would they write a fixed version number there?

**Comment by mwelche** [comment]
This issue has been resolved with 1.0.1. thanks!!
